base/strings/string.jl

```diff
@@ -469,7 +469,7 @@ function _length_continued_nonascii(
     return c
 end
 
-@inline function length_continued(s::String, first::Int, last::Int, c::Int)
+@assume_effects :terminates_locally @inline @propagate_inbounds @inline function length_continued(s::String, first::Int, last::Int, c::Int)
```

Comment:
double `@inline`

Reply:
Even When I fixed this I still got the error

Reply:
oh, sorry that was unrelated.